### PR TITLE
Updating cli docs for octo.exe

### PR DESCRIPTION
**Pull request by Octobob**
An automated update of the command line docs for octo.exe.

Created by TeamCity project 'Automated Documentation Updates::Update CLI Docs', build 40

### CODE REVIEW

File: docs/api-and-integration/octo.exe-command-line/push-metadata.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: push-metadata
+description: Pushes package metadata to Octopus Server.
+---
```
Comment (wordlee) [inline docs/api-and-integration/octo.exe-command-line/push-metadata.md]: To keep the sorting straight in the TOC:

```suggestion
position: 285
---
```

**Comment by slewis74** [comment]
I hadn't added this documentation page because the octo.exe command is really only for use by the build server extensions. We can add it for completeness if we have to, but I expect nobody would ever use it directly. Especially without knowing the JSON format of the file is requires. I would not bother with this page, or if we do go ahead with it include a link to the OctopusClient code repo for the object structure the file must contain.

**Comment by matt-richardson** [comment]
This gets auto-generated from the command line help, on the theory that if its worth having command line help, its worth having in the docs

**Comment by wordlee** [comment]
We shouldn't add anything to the docs unless we think it will be helpful. Matt can you exclude this page from being auto-generated?

**Comment by slewis74** [comment]
Thought more about this after a conversation with AppVeyor just now. We've been the ones build the build server plugins so far, but if we're not the ones building the build server plugin this doc has value.

**Comment by slewis74** [comment]
I think we should also add a link to https://github.com/OctopusDeploy/OctopusClients/blob/master/source/Octopus.Client/Model/PackageMetadata/OctopusPackageMetadata.cs in the OctopusClient code for the description about the file, so it's clear what JSON has to be in the file.

**Comment by wordlee** [comment]
I'm going to close this one as we have another PR that supersedes this one (I'll make the same edits in the new PR). And when the link has been added to OctopusClient code it'll get picked up next time OctoBob does his thing.

**Pull request closed without merging**